### DISCUSSION TITLE
功能建议

### DISCUSSION THREAD

**Discussion by Boris-code** [Boris-code/feaplat#3] [Ideas]
- [x] 监控面板区分用户角色，普通用户仅可查看自己的任务
- [x] 监控面板 携带任务名，友好化显示
- [x] 拉取代码支持指定分支
- [x] 管理员可分配项目给多个人
- [ ] 管理员身份时可查看项目及任务的创建人、参与人
- [x] 监控面板表字段支持筛选，解决字段太多UI拥挤的问题
- [ ] 项目支持依赖包隔离，类似虚拟环境功能
- [x] 取消心跳报警，直接显示爬虫容器状态
- [x] 优化调度，支持上千爬虫数同时启动
- [ ] 支持删除监控面板

**Comment by georgeJzzz** [comment]
- [x] 支持多种数据库去推送任务到redis， 例如目标库rabbitmq， mongodb
- [x] 自定义报警发送多个用户，或者指定用户

博哥nb 👍

**Reply by Boris-code** [reply]
亲，走错片场了，你说的是feapder吧，这是feaplat

**Reply by Boris-code** [reply]
你说的这俩已经支持了，下一版本就有

**Reply by georgeJzzz** [reply]
> 你说的这俩已经支持了，下一版本就有

好的  博哥nb

**Comment by chang-xiao-feng** [comment]
1、管理员可以分配项目给普通用户。
2、任务调度支持多选，同时启动 / 停止。
3、管理员创建的用户，不能查看授权码/密码。 可以设置SPIDER_ENV
4、已经删除的任务，爬虫监控-表名，不用在显示了吧。

**Comment by smile1847** [comment]
在创建项目时新增.env配置文件的上传，避免数据库等信息泄漏到github，docker run  --env-file .env 即可指定配置运行

**Comment by exlazybobby** [comment]
增加容器非root用户启动
